tests/Unit/Evolution/Initialization/Test_FunctionOfTimeFromFile.cpp

```diff
@@ -0,0 +1,138 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "tests/Unit/TestingFramework.hpp"
+
+#include <array>
+#include <cstddef>
+#include <string>
+#include <unordered_map>
+#include <vector>
+
+#include "DataStructures/DataBox/DataBox.hpp"
+#include "DataStructures/DataBox/DataBoxTag.hpp"
+#include "DataStructures/DataBox/Prefixes.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Domain/ElementId.hpp"
+#include "Domain/ElementIndex.hpp"
+#include "Domain/FunctionsOfTime/FunctionOfTime.hpp"
+#include "Domain/FunctionsOfTime/PiecewisePolynomial.hpp"
+#include "Domain/FunctionsOfTime/Tags.hpp"
+#include "Evolution/Initialization/FunctionOfTimeFromFile.hpp"
+#include "Evolution/Initialization/Tags.hpp"
+#include "Informer/InfoFromBuild.hpp"
+#include "Options/Options.hpp"
+#include "Options/ParseOptions.hpp"
+#include "Parallel/PhaseDependentActionList.hpp"
+#include "Utilities/FileSystem.hpp"
+#include "Utilities/Gsl.hpp"
+#include "Utilities/TMPL.hpp"
+#include "tests/Unit/ActionTesting.hpp"
+
+namespace {
+constexpr size_t dim = 2;
+using ElementIndexType = ElementIndex<dim>;
+
+template <typename Metavariables>
+struct component {
+  using metavariables = Metavariables;
+  using chare_type = ActionTesting::MockArrayChare;
+  using array_index = ElementIndexType;
+  using simple_tags = tmpl::list<Initialization::Tags::FuncOfTimeFile,
+                                 Initialization::Tags::FuncOfTimeSetNames>;
+  using phase_dependent_action_list = tmpl::list<
+      Parallel::PhaseActions<
+          typename Metavariables::Phase, Metavariables::Phase::Initialization,
+          tmpl::list<ActionTesting::InitializeDataBox<simple_tags>>>,
+      Parallel::PhaseActions<
+          typename Metavariables::Phase, Metavariables::Phase::Testing,
+          tmpl::list<Initialization::Actions::FunctionOfTimeFromFile>>>;
+};
+
+struct Metavariables {
+  using component_list = tmpl::list<component<Metavariables>>;
+  enum class Phase { Initialization, Testing, Exit };
+};
+}  // namespace
+
+SPECTRE_TEST_CASE("Unit.Evolution.FunctionOfTimeFromFile",
```

Comment:
Can you combine the two test cases by making the current test cases functions in an anonymous namespace and then calling them from the one `SPECTRE_TEST_CASE`? This will help reduce total test runtime

Reply:
Sure! Good to know that this shortens the test runtime